Update github-trigger-event-options.adoc (#9217)
circleci/circleci-docs: jekyll/_cci2/github-trigger-event-options.adoc

GitHub App triggers allow you to define the work to do (pipeline) when an event takes place (trigger). GitHub App triggers are an orchestration tool available in CircleCI. You can access GitHub App trigger options using the "Run on" menu when setting up a trigger. The following preset trigger options are available:

* *All pushes*: Trigger your pipeline on all pushes to your repo. All pushes corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event]. No pipelines are triggered on push events when a branch or tag are deleted. No pipeline is triggered when a branch is created.

> * *Tag pushes*: Trigger a pipeline on pushes to tags only. When choosing this option, don't forget to link:https://circleci.com/docs/workflows/#executing-workflows-for-a-git-tag[define tag filters in your config file].

Reviewer: ```suggestion
* *Tag pushes*: Trigger a pipeline on pushes to tags only. When choosing this option, you will also need to link:https://circleci.com/docs/workflows/#executing-workflows-for-a-git-tag[define tag filters in your config file].
```

* *PR opened*: Trigger a pipeline when a PR is opened. This applies also if the PR has been opened in draft. It does not apply if the PR is closed and then re-opened. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=opened#pull_request[GitHub docs for the pull request opened event].
* *PR merged*: Trigger a pipeline when a PR is merged only. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=closed#pull_request[GitHub docs for pull request closed].
* *PR marked ready for review*: Trigger a pipeline when a PR is taken out of draft mode and marked ready for review. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=review_requested#pull_request[GitHub docs for pull request review requested].
* *"run-ci" label added to PR*: Trigger a pipeline when a label "run-ci" is added to a PR. See the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=labeled#pull_request[GitHub docs for pull request labelled].

> [Note] 

Reviewer: ```suggestion
[NOTE] 
```

====

> Due to a bug, repeated pull request events of the same type do not result in build if he associated commit SHA has not changed. Let's take as an example a trigger configured with the option "PR marked ready for review": marking a PR as ready for review, then converting it to draft, then marking it as ready for review again will lead to only one pipeline to be triggered, not two. We are working to resolve this issue.

Reviewer: ```suggestion
Due to a bug, repeated pull request events of the same type do not result in a build if the associated commit SHA has not changed. As an example, consider a trigger configured with the option "PR marked ready for review". Marking a PR as ready for review, then converting it to draft, then marking it as ready for review again will lead to only one pipeline to be triggered, not two. We are working to resolve this issue.
```

====

> [Note] 

Reviewer: ```suggestion
[NOTE] 
```

====

> We plan to expand the available events and conditions to choose from. Let us know what additional triggers your team needs by filling out link:https://forms.gle/j7DC78CQNRmSBn159[this form], so we can prioritize adding them next.

Reviewer: ```suggestion
*Help us prioritize additional trigger options?* We are working to expand the available events and conditions for you to choose from. Let us know which additional triggers your team needs by filling out link:https://forms.gle/j7DC78CQNRmSBn159[this form].
```
